mistralrs-server/src/main.rs

```diff
@@ -251,10 +298,15 @@ async fn chatcompletions(
         let response = rx.recv().unwrap();
 
         match response {
-            Response::Error(e) => {
+            Response::InternalError(e) => {
                 MistralRs::maybe_log_error(state, &*e);
-                ChatCompletionResponder::Error(e)
+                ChatCompletionResponder::InternalError(e)
+            }
+            Response::ModelError(msg, response) => {
+                MistralRs::maybe_log_response(state, &response);
```

Comment:
I think this should be a call to `maybe_log_error`.

Reply:
Yeah, my bad